ui/components/app/gas-details-item/gas-details-item.js

```diff
@@ -32,14 +32,16 @@ const GasDetailsItem = ({ userAcknowledgedGasMissing = false }) => {
     return null;
   }
 
+  const loader = <LoadingHeartBeat estimateUsed={estimateUsed} />;
```

Comment:
[Q] What benefit does it serve assigning this to a variable vs having 3 instances of the component? genuinely curious, no changes are required.

Reply:
Done 👍